eduvpn/actions/select.py

```diff
@@ -19,7 +21,7 @@
 
 
 # ui thread
-def select_profile(builder, verifier, lets_connect):
+def select_profile(builder, verifier, lets_connect):  # type: (Gtk.builder, str, bool) -> Metadata
```

Comment:
ik denk dat deze ook > 120 is

Reply:
je kan de type comment toch op een nieuwe regel neerzetten? doe je op andere plekken ook.